packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -73,8 +73,10 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     this._id = getId('TextField');
     this._descriptionId = getId('TextFieldDescription');
 
+    this._latestValue = props.value || props.defaultValue || '';
```

Comment:
Can you add:

1. a change file
2. a unit test to validate what you're changing?

Reply:
Thanks for the feedback. Added the changes.
It's the first test I contribute to the repo so please let me know if it needs adjustments :)

Reply:
Thank you @htryggva !